pyproject.toml

```diff
@@ -9,15 +9,34 @@ license = "MIT"
 readme = "README.md"
 keywords = ["graphql", "api", "django"]
 classifiers = [
-  "Topic :: Software Development :: Libraries",
-  "Topic :: Software Development :: Libraries :: Python Modules",
+  "Environment :: Web Environment",
+  "Intended Audience :: Developers",
+  "License :: OSI Approved :: MIT License",
+  "Operating System :: OS Independent",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Framework :: Django",
+  "Framework :: Django :: 3.2",
+  "Framework :: Django :: 4.0",
 ]
 
 [tool.poetry.dependencies]
 python = ">=3.7,<4.0"
 Django = ">=3.0"
 strawberry-graphql = ">=0.109.0"
 
+# docs-deps
+mkdocs = { version = "^1.3.0", optional = true}
+mkdocs-markdownextradata-plugin = {version = "^0.2.5", optional = true}
+mkdocs-material = {version = "^8.3.8", optional = true}
+mkdocs-minify-plugin =  {version = "^0.5.0", optional = true}
+pymdown-extensions = {version = "^9.5", optional = true}
+markdown = {version = "^3.3.7", optional = true}
```

Comment:
Those should probably be added to dev-dependencies

Reply:
Also there will prob be depandancy issues and python version incompatibility.